test/fw/ptl/utils/pbs_testsuite.py

```diff
@@ -1474,9 +1487,15 @@ def tearDown(self):
 
         for sched in self.scheds:
             self.scheds[sched].cleanup_files()
+        self.server.del_hooks_nodes_qs()
```

Comment:
Interesting, this will add some part of "revert" action to tearDown() as well. We will delete hooks, nodes, queues and reservations in this function and then load the "default" configuration, where the default could either be PTL's default, or the user's provided configuration.
Ideally, we should do the following:
- split the "revert" functions into two parts: delete and load, where delete would be common but the load function will either load the PTL's default, or the user provided configuration.

But, since this PR has been hanging around for a long time, if you don't want to do the refactoring, then I suggest that you only call "self.server.del_hooks_nodes_qs()" and "self.server.load_configuration(self.saved_file)" if "self.use_cur_setup" is True. Similarly, only call "self.server.save_configuration(path, 'w')" in setUp() if "self.use_cur_setup" is True. This will mean that when "self.use_cur_setup" is not True, PTL will behave exactly like before.